tests/llm/test_ask_holmes.py

```
@@ -149,12 +139,13 @@
         assert scores.get("correctness", 0) >= test_case.evaluation.correctness
 
 
-def ask_holmes(test_case: AskHolmesTestCase) -> LLMResult:
+def ask_holmes(test_case: AskHolmesTestCase, eval: Span) -> LLMResult:
     run_live = load_bool("RUN_LIVE", False)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)